### PR TITLE
Fix apt usage and improve classic docs

### PR DESCRIPTION
**Pull request by jawn-smith**
This addresses two things:

If the package version in /var/lib/dpkg/status was newer than what is available on the target SERIES, errors would occur. Using -o Dir::State::status=<empty_file> avoids that and only considers versions present in the apt sources list.

Docs for classic were pretty outdated. We need to use make to prime the gadget snap rather than classic, and the psplash stuff doesn't actually seem to apply to classic images.

I know it's generally thought of as not-best-practice to have a different README per branch, but that's because branches are meant to be ephemeral in most cases. Since this classic branch is meant to stay around, I think it makes sense to have a separate README.

### CODE REVIEW

**Comment by waveform80** [comment]
Excellent, this works nicely. Any chance you can rebase this to split the README into a separate commit? (this'll just make it easier for me to cherry-pick the relevant bits across our myriad branches!)

**Review by sil2100 — APPROVED** [review]
This is good!

Talking about README... maybe we should actually get all this and create one global README.md that will have all the needed info: for classic, for UC20, for UC18 etc. - all subsections. Maybe also mentioning what branches mean what!

**Comment by waveform80** [comment]
> This is good!
> 
> Talking about README... maybe we should actually get all this and create one global README.md that will have all the needed info: for classic, for UC20, for UC18 etc. - all subsections. Maybe also mentioning what branches mean what!

Yes, there's a bunch of README changes in the core branches so once this is split out I'll merge it and then see if I can merge README changes from the various branches (and add some docs about what those branches are for -- actually that's what the core README changes are about)

**Comment by jawn-smith** [comment]
@waveform80 done, separate commits.

**Comment by waveform80** [comment]
Brilliant, thanks -- I'll get on with some changes on top of this and then see about merging the relevant bits across the various branches.

**Pull request merged**
